apps/cnquery/cmd/providers.go

```diff
@@ -51,9 +52,17 @@ var listProvidersCmd = &cobra.Command{
 var installProviderCmd = &cobra.Command{
 	Use:    "install <NAME>",
 	Short:  "Install or update a provider.",
+	Args:   cobra.MinimumNArgs(1),
```

Comment:
that does not work with --file and --url

Reply:
I changed that to `ExactArgs(1)`, so we can only pass in the provider's name, i don't think the --file and --url flags count towards that number, those are flags and not args